### PR TITLE
Add sort keys for "natural" landuse kinds

### PR DESCRIPTION
**Pull request by zerebubuth**
Insert `natural_wood`, `natural_forest` and `natural_park` into the middle of the sort ranking, bumping up all the values above them.

Connects to #1096.

16 total new:

- [x] natural_wood
- [x] natural_forest
- [x] natural_park
- [x] airfield
- [x] danger_area
- [x] naval_base
- [x] port_terminal
- [x] ferry_terminal
- [x] container_terminal
- [x] shipyard
- [x] wharf
- [x] quay
- [x] boatyard
- [x] range
- [x] wetland
- [x] embankment (lines)

### CODE REVIEW

**Comment by nvkelso** [comment]
Please update for the following issues, too:
 
- #1569 Wetland should sort above water bug priority medium
- #1574 Set better sort_rank on embankment lines bug priority medium
- #1588 Set sort_order for quay, wharf, other new landuse kinds bug priority medium

**Comment by zerebubuth** [comment]
Added all those others - except for `boatyard` and `range`. Those are POIs only, and we don't generate landuse polygons for them at the moment. Would you like me to add landuse polygons for those, or are they fine as POIs?

**Comment by nvkelso** [comment]
Ooops, yes those should also be landuse polygons. Good catch!

File: spreadsheets/sort_rank/water.csv [hunk 1]
```diff
@@ -1,16 +1,16 @@
 kind,boundary,sort_rank
-river,,201
```
Comment (nvkelso) [inline spreadsheets/sort_rank/water.csv]: What does the `-` mean here?

File: spreadsheets/sort_rank/landuse.csv [hunk 2]
```diff
@@ -18,85 +19,97 @@ groyne,*,225
...
+taxiway,*,72
+aerodrome,*,70
+airfield,*,69
+recreation_track,*,68
+caravan_site,*,67
+recreation_ground,*,66
+quay,*,65
+wharf,*,65
+port_terminal,*,64
+ferry_terminal,*,64
```
Comment (nvkelso) [inline spreadsheets/sort_rank/landuse.csv]: Can't share sort values otherwise it creates flickr no Tangram ES... so this and quay and possibly others need to offset away from each other.

File: spreadsheets/sort_rank/landuse.csv [hunk 3]
```diff
@@ -1,4 +1,5 @@
 kind,geometrytype,sort_rank
+embankment,*,274
```
Comment (nvkelso) [inline spreadsheets/sort_rank/landuse.csv]: Seems like this should go below power_line, more just under fence?

File: spreadsheets/sort_rank/water.csv (hunk 1 above)
Comment (zerebubuth) [reply spreadsheets/sort_rank/water.csv]: It means `boundary` is not set. The empty match in `river,,201` translates to `kind == 'river' and boundary == ''`, which never matched (`boundary` is either `None` or `True`), so all water features were getting a `sort_rank` of 200.

File: spreadsheets/sort_rank/landuse.csv (hunk 2 above)
Comment (zerebubuth) [reply spreadsheets/sort_rank/landuse.csv]: I've re-arranged them in ffe23bd to make them unique. Also added a test, so that we'll get a test failure if a duplicate is added in future.

File: spreadsheets/sort_rank/landuse.csv (hunk 3 above)
Comment (zerebubuth) [reply spreadsheets/sort_rank/landuse.csv]: OK. Fixed in ffe23bd.

**Comment by nvkelso** [comment]
The following two kind values are new to the landuse layer and need to be documented:

- `boatyard`
- `range`
